feature-group-definitions/navigation-activation.yml

```diff
@@ -0,0 +1,2 @@
+name: Navigation Activation
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#navigation-activation-interface
```

Comment:
Searching for "NavigationActivation" in the spec, the `PageSwapEvent` interface is also related, by having an `activation` member. But that's not in https://github.com/mdn/browser-compat-data/pull/22280, was it not shipped together?

Reply:
No, it hasn't shipped yet. `PageSwapEvent` builds on top of the `NavigationActivation` feature.

Reply:
I see, thanks for confirming!